integration/experiment/gpu_frequency_sweep/gpu_frequency_sweep.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python3
+#
+#  Copyright (c) 2015 - 2022, Intel Corporation
+#  SPDX-License-Identifier: BSD-3-Clause
+#
+
```

Comment:
Make sure to add [co-authors](https://docs.github.com/en/pull-requests/committing-changes-to-your-project/creating-and-editing-commits/creating-a-commit-with-multiple-authors) for anything re-used from other pull requests (ex: [gpu_frequency_sweep.py](https://github.com/geopm/geopm/blob/2ddf7e6f7146b2de0bd6f9838a416090a9d4f434/integration/experiment/gpu_frequency_sweep/gpu_frequency_sweep.py) from #2024)

Reply:
I'd attribute Sari as an external collaborator for starting the scripts but wouldn't worry about going back through the full chain of internal team collaborators.  A quick diff should show similarity % if it helps!